CHANGELOG.mkd

```diff
@@ -2,11 +2,15 @@ CHANGELOG
 =========
 
 Unreleased
+----------
 
-- Don't purge root when using `puppetfile install`. [#1084](https://github.com/puppetlabs/r10k/issues/1084)
-
+2.6.9
 ----
 
+### Bug Fixes
```

Comment:
I think on 3.x we've stopped adding these tags, but I'm fine if we want to put it in here.

Reply:
ah yeah, I can remove it, since we also moved to adding the `-` to match mainline.